lms/envs/test.py

```diff
@@ -602,4 +602,8 @@
 NODEBB_RETRY_DELAY = 60
 NODEBB_ENDPOINT = "http://local.philanthropyu.org:4567"
 # replace NODEBB_MASTER_TOKEN with value from your setup
-NODEBB_MASTER_TOKEN = "5780dbb9-c042-4290-a483-4b84c3b161ee"
+NODEBB_MASTER_TOKEN = "b4f6c9c3-d778-4b47-bdbc-73535cfdbe3b"
+MANDRILL_API_KEY = "VmO_YjiGNJOJ6KukuRnhZA"
+MAILCHIMP_API_KEY = "e6c22b333c17e79494ae5e307e3364e5-us12"
```

Comment:
Get these keys from constants file

Reply:
Done. These keys have been disabled and new keys have been generated and distributed.